index.js

```diff
@@ -85,14 +85,41 @@ export async function runASTAnalysisOnFile(pathToFile, options = {}) {
 
 export const CONSTANTS = {
```

Comment:
Do we want to keep exporting `export const CONSTANTS = {
  Warnings: ...
}`

Instead exporting directly the warnings object

Reply:
Yes